plugins/eventMacro/eventMacro/Runner.pm

```diff
@@ -660,7 +660,7 @@ sub define_next_valid_command {
 		# Initial 'if'
 		######################################
 		} elsif ($self->{current_line} =~ /^if\s/) {
-			my ($condition_text, $post_if) = $self->{current_line} =~ /^if\s+\(\s*(.*)\s*\)\s+(goto\s+.*|call\s+.*|stop|{|)\s*/;
+			my ($condition_text, $post_if) = $self->{current_line} =~ /^if\s+\(\s*(.*)\s*\)\s*(goto\s+.*|call\s+.*|stop|{|)\s*/;
```

Comment:
If were going to allow

if (x){
shouldn't we allow
if(x){
?

I mean like
`/^if\s*\(\s*(.*)\s*\)\s*(goto\s+.*|call\s+.*|stop|{|)\s*/`

Reply:
no we need that for future to separate possible sub if.... from if block

Reply:
But `/^if\s*\(\s*(.*)\s*\)\s*(goto\s+.*|call\s+.*|stop|{|)\s*/` wouldn't match sub ifsomething (note the `^`)

Reply:
i mean if() <- function if () <- condition